modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-embed-into-rpm-ostree.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-creating-ostree-iso_{context}"]
+= Creating the {op-system-ostree-first} image
+
+Use the following procedure to create the ISO. The {op-system-ostree} Installer image pulls the commit from the running container and creates an installable boot ISO with a Kickstart file configured to use the embedded OSTree commit.
+
+.Prerequisites
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+* You have root-user access to your build host.
+* You have the `podman` tool.
+
+.Procedure
+
+. Start an `ostree` container image build by running the following command:
++
+[source,terminal]
+----
+$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/9/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
+----
++
+This command also returns the identification (ID) of the build for monitoring.
+
+. You can check the status of the build periodically by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli compose status
```

Comment:
Would it be appropriate to add the watch command to the beginning of this line  ? It will poll the command based on a time interval. 

_watch sudo composer-cli compose status_

Reply:
In the description, we tell the user to check the status of the build `periodicaly`. While `watch` has the same effect, I think we may want to avoid it here because then, we need to explain what it does, how to quit it, etc.
Any average user should automatically fall back to using `watch` when asked to check `periodically`, I think

Reply:
That makes sense. Introducing something like `watch` might cause an unintended issue.